neuralmonkey/runners/label_runner.py

```diff
@@ -40,9 +44,15 @@ def loss_names(self) -> List[str]:
 
 class LabelRunExecutable(Executable):
 
-    def __init__(self, all_coders, fetches, vocabulary, postprocess):
+    def __init__(self,
+                 all_coders,
+                 fetches,
+                 num_sessions,
+                 vocabulary,
+                 postprocess):
```

Comment:
type hints

Reply:
postprocess je jakeho typu?

Reply:
Obecně `Callable[[List[Any]], List[Any]]` plus mínus závorka. Ze seznamu čehokoliv do seznamu čehokoliv. Bude se lišit u každého runneru, ale to nech zatim bejt.